.gitignore

```diff
@@ -2,4 +2,29 @@ node_modules
 .env
 dist
 .DS_Store
-.vscode
\ No newline at end of file
+.vscode
```

Comment:
This whole build process looks painful as hell. Super impressed you did it though, you'll have to show me sometime!